src/hooks/useMarketForm.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+
+const useMarketoForm = (munchkinId, id, pubishableKey) =>
+  useEffect(() => {
+    window.MktoForms2.loadForm('//app-abj.marketo.com', munchkinId, id);
+
+    const pollForDefinition = function (scope, varname, callback) {
+      if (typeof scope[varname] !== 'undefined') {
+        return callback();
+      }
+      const interval = setInterval(function () {
+        if (typeof scope[varname] !== 'undefined') {
+          clearInterval(interval);
+          callback();
+        }
+      }, 250);
+    };
+    const script = document.createElement('script');
+    script.src = 'https://marketo.clearbit.com/assets/v1/marketo/forms.js';
+    script.async = true;
+    script.setAttribute('data-clearbit-publishable-key', pubishableKey);
+    script.onerror = function () {
+      // eslint-disable-next-line no-console
+      console.log('Clearbit Form JS unable to load');
```

Comment:
We do have New Relic browser installed on the dev site. Would it be better to maybe send an event through that? That would make it easier to detect when the form is unable to load so we can look into this if we see it happening a lot.

Reply:
Good idea, I'll create an issue for this, being that I have a couple of questions for you about adding it to the NR browser.